app/views/handlers/question.py

```diff
@@ -19,14 +19,18 @@
 
 class Question(BlockHandler):
     @staticmethod
-    def _has_redirect_to_list_add_action(answer_action):
-        return answer_action and answer_action["type"] == "RedirectToListAddBlock"
+    def _has_redirect_to_list_add_action(answer_action: Mapping | None) -> bool:
+        return (
+            answer_action is not None
```

Comment:
Did this need changing?. Previous one would pass with empty dict, this would error on KeyError.

Reply:
Mypy didn't like it but instead of changing the code I could have written the following line actually. 

```return bool(answer_action and answer_action["type"] == "RedirectToListAddBlock")```